doc/project/release_process.rst

```diff
@@ -73,8 +128,8 @@ Stabilization Phase
 *******************
 
 Over the next weeks and depending on the release milestone, only stabilization,
-cosmetic changes, tests, bug and doc fixes are allowed (See :ref:`table
-<release_milestones>` below).
+cosmetic changes, tests, bug and documentation fixes and enhancements are
```

Comment:
```suggestion
cosmetic changes, tests, bug and documentation changes are
```
Otherwise somebody will read this as "all enhancements anywhere (not just in docs) are allowed"

Reply:
It'd need to be "bug fixes and documentation changes". That said, you already have "cosmetic changes" earlier in the sentence, so it begs the question why it's not grouped there (i.e. "cosmetic and documentation changes"). I wonder if this would be more clearly expressed as a bullet point list instead.

Reply:
Note we already have the table, so maybe we are trying to be too explicit and repetitive here.